sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -52,6 +52,8 @@ case class ExprCode(var code: String, var isNull: String, var value: String)
  */
 class CodegenContext {
 
+  var conf: CatalystConf = null
```

Comment:
this is really hacky -- i'd put this in the constructor and make it a val rather than a var. and maybe we can create a CodegenConf instead of reusing CatalystConf?